core/src/test/java/org/infinispan/api/mvcc/repeatable_read/WriteSkewTest.java

```diff
@@ -57,6 +57,7 @@ public void setUp() {
       // The default cache is NOT write skew enabled.
       cacheManager = TestCacheManagerFactory.createCacheManager(configurationBuilder);
       configurationBuilder.locking().isolationLevel(IsolationLevel.REPEATABLE_READ);
+      configurationBuilder.clustering().hash().groups().enabled();
```

Comment:
@rvansa This fails with trace logging enabled, because it forces the component registry to create a `DistributionManagerImpl`, and its `start()` methods throws a NPE if the cache manager is not clustered. Do you really need grouping in this class?